docs/docs/hive.md

```diff
@@ -213,7 +195,7 @@ SET iceberg.catalog.glue.lock.table=myGlueLockTable;
 
 ## DDL Commands
 
-Not all the features below are supported with Hive 2.3.x and Hive 3.1.x. Please refer to the
+Not all the features below are supported with Hive 3.1.x. Please refer to the
 [Feature support](#feature-support) paragraph for further details.
```

Comment:
Again, we can remove the `feature-support` table, and everything related to Hive 3.1.x, and below.
We should only talk about things supported by Hive4